app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -18,18 +18,47 @@ class GetUspsProofingResultsJob < ApplicationJob
 
   discard_on GoodJob::ActiveJobExtensions::Concurrency::ConcurrencyExceededError
 
+  def enrollment_attributes(enrollment)
+    attributes = {
+      enrollment_code: enrollment.enrollment_code,
+      enrollment_id: enrollment.id,
+      minutes_since_last_status_check: enrollment.minutes_since_last_status_check,
+      minutes_since_last_status_update: enrollment.minutes_since_last_status_update,
+    }
+    attributes[:minutes_to_completion] = enrollment.minutes_to_completion if enrollment.complete?
```

Comment:
would it be ok to have a key but a nil value? if we inlined this then we wouldn't need to assign a variable for the hash 🤷 
```suggestion
      minutes_to_completion: enrollment.complete?.presence && enrollment.minutes_to_completion,
    }
```